js/script.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+(function(){ 
+	/* W kodzie HTML i CSS dodaliśmy style dla prostego modala, który będzie zawsze wyśrodkowany w oknie. 
+	
+	Teraz wystarczy napisać funkcję otwierającą modal:
+	*/
```

Comment:
comments in your code, should be in english

Reply:
@matwjablonski  It is a Kodilla's code, so I didn't change the comments language. I can delete them.

Reply:
so delete them ;) remember that your code should be always clean when you send something to pull request ;)